README.md

```diff
@@ -101,6 +101,11 @@ will be replaced with the newly built image if `os_images_upload` is set to `Tru
 
 `os_images_dib_venv`: Path to virtualenv in which to install DIB to build images.
 
+`os_images_cleanup`: __CAUTION__ (optional) boolean flag indicating whether or not the images should be deleted.
```

Comment:
Should we not add a note here saying, when images are deleted it might cause problems for servers using that image, and you should instead look at hiding the image?

Reply:
Actually, yeah, I see is_hidden isn't exposed as a parameter in here: https://github.com/openstack/ansible-collections-openstack/blob/master/plugins/modules/image.py#L462

Reply:
Added more clarification.